src/rest/resources/Messages.ts

```diff
@@ -26,11 +27,14 @@ export class MessagesResource {
         : {
             embed: message.render()
           }
-    }
-    if (typeof message === 'string') {
+    } else if (typeof message === 'string') {
```

Comment:
Better way to keep this inline rather than an if?

Reply:
What do you mean by this?

Reply:
Rather than what you have, something like

```ts
if ([types].includes(message)) {
  message = {
    content: Util.inspect(message)
  }
}```
and just include "string" in the `types` as it'll be inspected fine like any of the others.

Reply:
Ah, I see now. I will fix that.